pkgs/development/python-modules/flatbuffers/default.nix

```
@@ -0,0 +1,20 @@
+{ stdenv
+, buildPythonPackage
+, pkgs
+}:
+
+let
+  flatbuf = pkgs.flatbuffers;
```

You shouldn't be passing `pkgs` in a derivation like this.
```suggestion
, flatbuffers
}:

let
  flatbuf = flatbuffers;
```

Tested on my NixOS machine, works fine.

---

This yields an infinite recursion on my machine. I'm calling this with the following:
```
nix-shell -I nixpkgs=../nixpkgs -p 'python3.withPackages(ps: with ps; [ flatbuffers ])' --show-trace
```

Can you further investigate or explain why I would see an infinite recursion here?

---

For this to work, we have to be explicit about what we're passing to the derivation in `pkgs/top-level/python-packages.nix`. This should be resolved in the latest commit.